.circleci/config.yml

```diff
@@ -589,10 +593,18 @@ commands:
             "pnpm-lock.yaml" }}
       - run:
           name: Setup Dependencies
-          command: pnpm install --frozen-lockfile
+          command: |
+            sudo apt update -y
+            sudo apt install libx11-dev libpng-dev libjpeg-dev
```

Comment:
Just for my understanding, why were these additions of `libx11-dev`, `libpng-dev`, and `libjpeg-dev` needed? Is it because the new `cimg` images are trimmed down from the older `circleci` images. (That's at least what I'm gleaning from [this CircleCI article](https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic/))

Reply:
Yep! They were not pre-installed on the new `cimg` images.

Reply:
Nice! Thanks for confirming